.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: Build
+
+on: [push]
```

Comment:
Looking at this again before merging, could you change this to `push, pull_request`? I don't think this will run the tests on pull requests opened on this repo without that.